suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -194,6 +194,18 @@ export const selectAccountByDescriptorAndNetworkSymbol = memoizeWithArgs(
     },
 );
 
+export const selectAccountKeyByDescriptorAndNetworkSymbol = memoizeWithArgs(
```

Comment:
We cannot use the key directly because this information is not inside of `EventNotification` object. The shape is as follows: 
```
{
        context: 'event',
        descriptor:
            'vpub5ZjRPuuMiEQnbwEDi9jtH1FaJMajZW78uZ1t3RJXKhxyMoTnPraKwGxiDo9SguDYvSieqjoLJxW5n2t9156RR1oeqRnURuftNZTzejBc4pa',
        device: undefined,
        formattedAmount: '0.00055 TEST',
        id: 1678029249638,
        symbol: 'test',
        txid: '2a291100058438b66b479f2d7f474a9bc7dd36b7988220af5422a99932938f00',
        type: 'tx-received',
}        
```

for that reason we first need to get accountKey from descriptor and networkSymbol. When we have the accountKey then we are able to get transaction.

The operations then  are:
[descriptor, networkSymbol] -> accountKey
[accountKey, txid] => transaction object